h2/src/main/org/h2/command/Parser.java

```diff
@@ -6084,6 +6085,7 @@ private Prepared parseAlterTable() {
                     Column column = table.getColumn(columnName);
                     columnsToRemove.add(column);
                 } while (readIf(","));
+                readIf(")"); // For Oracle compatibility - close bracket
```

Comment:
you need a boolean if you read the first bracket, and then this line should be read(")") not readIf, otherwise we could accept some dodgy syntax :-)

Reply:
Yes, fix is on the way..